src/supervisor.tex

```diff
@@ -805,6 +805,10 @@ \subsection{Supervisor Address Translation and Protection ({\tt satp}) Register}
 and if {\tt satp} is written with an unsupported MODE, the entire write has
 no effect; no fields in {\tt satp} are modified.
 
+When memory translation is active, instruction fetches and memory loads and stores
+from S-mode and U-mode go through the translation mechanism described in
+Section~\ref{sv32algorithm}.
+
```

Comment:
This is only true for currently defined MODEs.  I would remove this paragraph and instead augment the table of MODEs to have pointers to the various SvXX sections (e.g. Page-based 32-bit virtual addressing (see Section~\ref{sec:sv32}))